lib/schlepper/process.rb

```diff
@@ -106,7 +106,12 @@ def say(*); end
       if status
         ActiveRecord::Base.connection.execute <<-SQL
           INSERT INTO schlepper_tasks (version, owner, description, completed_at)
-          VALUES (#{task.version_number}, #{ActiveRecord::Base.connection.quote(task.owner)}, #{ActiveRecord::Base.connection.quote(task.description)}, #{ActiveRecord::Base.connection.quote(Time.now.to_s(:db))});
+          VALUES (
+            #{task.version_number},
+            #{ActiveRecord::Base.connection.quote(task.owner)},
+            #{ActiveRecord::Base.connection.quote(task.description.mb_chars.compose.limit(256))},
```

Comment:
👍 mb_chars is what we want.

🤔 if it's worth it to expand the width of the description column

Reply:
Probably not worth it; adds a migration to every client for little value.  I'm hoping that the guidance to stay <= 256 bytes (in the template.rb.erb) will help people get enough description into the space they have.

Reply:
@RoryO @cobbr2  It would be also nice to `puts 'WARNING: description truncated'` so they feel little bit guilty for not following procedure ;)

Reply:
(I hope I didn't scare you too much ;))

Reply:
:wave: